src/UIComponents/ColumnInspector.jsx

```diff
@@ -136,7 +153,8 @@ class ColumnInspector extends Component {
 export default connect(
   state => ({
     currentColumnData: getCurrentColumnData(state),
-    rangesByColumn: getRangesByColumn(state)
+    rangesByColumn: getRangesByColumn(state),
+    hideSpecifyColumns: state.mode.hideSpecifyColumns
```

Comment:
This hits an exception when `mode` is `null` which happens in our default standalone invocation.

Reply:
(We guard against it [here](https://github.com/code-dot-org/ml-playground/pull/83/files#diff-6e02291d2b49a002990985f9cb58187da3ad794009887859ccce802714a852ddL32), for example.)

Reply:
done! Thanks for the tip.